packages/turbo/src/VisitableView.tsx

```diff
@@ -129,6 +131,7 @@ const VisitableView = React.forwardRef<RefObject, React.PropsWithRef<Props>>(
           ref={visitableViewRef}
           url={props.url}
           sessionHandle={sessionHandle}
+          webviewDebuggingEnabled={webviewDebuggingEnabled}
```

Comment:
We use `webView` in most other places. So maybe it would be better to have 
```suggestion
          webViewDebuggingEnabled={webViewDebuggingEnabled}
```

Reply:
> Maybe we can always enable debugging webview when app is in debugging mode
> ...
> This API makes it easy to set debugging enabled for production builds, which might not be desired option.

It's definitely a better idea.